src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlQueryMetadataCache.cs

```diff
@@ -282,7 +300,7 @@ private String GetCacheLookupKeyFromSqlCommand(SqlCommand sqlCommand)
             // Return null if we have no connection.
             if (connection == null)
             {
-                return null;
+                return Tuple.Create<string, string>(null, null);
```

Comment:
Is it possible to avoid the empty tuple creation here and just return null instead?

Reply:
Done.